.github/workflows/pull-request.yaml

```diff
@@ -41,6 +41,6 @@ jobs:
         uses: goreleaser/goreleaser-action@v2
         with:
           version: latest
-          args: --debug --snapshot
```

Comment:
Had to modify the arg, because the current goreleaser v2 [failed](https://github.com/canonical/cos-tool/actions/runs/13824503903/job/38676895259#step:4:19):

```
 /opt/hostedtoolcache/goreleaser-action/2.7.0/x64/goreleaser --debug --snapshot
  ⨯ command failed                                   error=unknown flag: --debug
```